configs/application/config.json

```diff
@@ -58,7 +61,7 @@
 			"unknownComponent": "nonConfiguredComponent"
 		},
 		"assimilation": {
-			"useOpenFinSpawn": false,
+			"useFeaSpawn": false,
```

Comment:
In assimilation window manager we're still using useOpenFinSpawn. This will cause that code to break

Reply:
Not in 5.0.0

Reply:
https://github.com/ChartIQ/finsemble/commit/c1f36d12d96aeaac29b9854d9c0ef04bbfdb44df

Reply:
Cool!